Emby.Server.Implementations/SocketSharp/WebSocketSharpRequest.cs

```diff
@@ -100,46 +98,46 @@ internal static ReadOnlySpan<char> CheckBadChars(ReadOnlySpan<char> name)
                 switch (crlf)
                 {
                     case 0:
-                    {
-                        if (c == '\r')
-                        {
-                            crlf = 1;
-                        }
-                        else if (c == '\n')
-                        {
-                            // Technically this is bad HTTP.  But it would be a breaking change to throw here.
-                            // Is there an exploit?
-                            crlf = 2;
-                        }
-                        else if (c == 127 || (c < ' ' && c != '\t'))
                         {
-                            throw new ArgumentException("net_WebHeaderInvalidControlChars", nameof(name));
-                        }
+                            if (c == '\r')
```

Comment:
This looks over-indented

Reply:
Nice catch, this should have been a 1 line commit but apparently I made a mistake. I will revert these changes.

By the way, this `CheckBadChars` method should probably be taken care off too.

Reply:
Ok I understand now. My editor (vscode with the provided workspace) indents cases like this:
```
case 0:
    {
```

So the indentation is in fact coherent. 
Is that wrong ?

(my diff view in vscode wasn't configured to show changes in indentation so I missed it before commit)

Reply:
Ok, I removed the useless curly brackets. Indent feels more natural now.